### ISSUE TITLE
Training is too GPU - power hungry

### ISSUE THREAD

**Issue by redasu** [cardwing/Codes-for-Lane-Detection#111]
While using gtx 1070 8gb (over 6gb free) I can only train with batch_size=1 which seems ridiculous. Maybe code is somewhat not optimized or I should expect these kind of drawbacks because of my gpu? Any suggestions? Even setting MEMORY_FRACTION = 0.95 does not help...

**Comment by cardwing** [comment]
You may need to check if there is anything wrong since even using one GPU with 12 GB graphic memory we can train the model with a batch size of 8. Besides, you may also try our ENet-Label-Torch model which is extremely lightweight and can achieve around 68.8 in CULane testing set.

**Comment by Leviou** [comment]
what should I do when I use the CULane dataset in SCNN-Tensorflow?  I cpoy the list/train_gt.txt&val_gt,txt.  to the dataset path,but I always get the Error not found :xxx/xxx.MP4/xxx.jpg
For example,Not found:driver_23_30frame/05170932_0736.MP4/02780.jpg. but I am sure that dataset contains the folder
